libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt

```diff
@@ -60,7 +60,7 @@ object FeedbackEvent {
     const val INACCURATE_GPS = "inaccurate_gps"
 
     /**
-     * Feedback type *incorrect visual guidance*: wrong visual guidance
+     * Feedback type *looks incorrect guidance*: wrong visual guidance
```

Comment:
Should this say “looks incorrect” instead of “looks incorrect guidance”?

Reply:
hm, should be